tasks/config/platforms.js

```diff
@@ -32,11 +32,11 @@ module.exports = function (grunt) {
     let rpmPath;
     if (name.match('linux')) {
       let debArch = name.match('x64') ? 'amd64' : 'i386';
-      debName = `kibana_${version}_${debArch}.deb`;
+      debName = `kibana-${version}-${debArch}.deb`;
       debPath = resolve(rootPath, `target/${debName}`);
 
-      let rpmArch = name.match('x64') ? 'x86_64' : 'i386';
-      rpmName = `kibana-${version.replace('-', '_')}-1.${rpmArch}.rpm`;
+      let rpmArch = name.match('x64') ? 'x86_64' : 'i686';
```

Comment:
we'll have to update https://github.com/epixa/kibana/blob/f7dc301d8ba72b00fc4754630bf970c017e064e8/tasks/build/os_packages.js#L20 too.  Might be a better way to centralize the architecture, we can move it to `tasks/config/packages.js`